<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="4">
     <widget class="QPushButton" name="Button_ans_pai">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ans</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="Button_del">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DEL</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="Button_add">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button_tan">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button_4">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button_0">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="QLineEdit" name="lineEdit_answer">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1400</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Button_shift">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>shift</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="Button_6">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Button_point_">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="Button_sub">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Button_right">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button_1">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Button_sin">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="Button_mul">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="Button_9">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Button_5">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6" rowspan="2">
     <widget class="QPushButton" name="Button_enter">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>400</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="Button_left">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="Button_3">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="Button_7">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button_cos">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Button_2">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="Button_div">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="Button_8">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Button_sqrt_e">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="Button_ac">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button_pow_ln">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="7">
     <widget class="QLineEdit" name="lineEdit_text">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1400</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
